src/main/java/frc/robot/subsystems/Intake/IntakeConstants.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.subsystems.Intake;
+
+import com.ctre.phoenix.motorcontrol.InvertType;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+public class IntakeConstants {
+    private static final int INTAKE_MOTOR_ID = 5;
+
+    static final WPI_TalonSRX INTAKE_MOTOR = new WPI_TalonSRX(INTAKE_MOTOR_ID);
```

Comment:
No need to type intake if it's already known, and it's the only motor in the class
Just name MOTOR( same with ID)